invenio_app_ils/ext.py

```diff
@@ -24,6 +27,8 @@ def __init__(self, app=None):
     def _init_app(self, app):
         """Flask application initialization."""
         self._init_config(app)
+        _editor_blueprint = create_editor_blueprint(app)
```

Comment:
Can you please remove this due to: https://github.com/inveniosoftware/invenio-records-editor/pull/5